wp-content/themes/chrisdinh-photography/resources/scripts/components/splide.js

```diff
@@ -1,29 +1,33 @@
-import { Splide } from "@splidejs/splide";
+import { Splide, SplidePagination } from "@splidejs/splide";
 
 const initiateSplideSlider = ( selector = '.splide' ) => {
   const sliders = document.querySelectorAll(`${selector}`);
-
   if ( sliders.length > 0 ) {
     sliders.forEach( slider => {
 
       const sliderSettings = setSliderSettings( slider );
       const sliderId = slider.id;
 
       if ( sliderId ) {
-        new Splide(`#${sliderId}`, sliderSettings).mount();
+        const splide = new Splide(`#${sliderId}`, sliderSettings).mount();
+
+        if (slider.dataset.customPagination === 'true') {
+          applyCustomPagination(splide);
+        }
       }
     });
   }
 };
 
 const setSliderSettings = ( slider ) => {
   const sliderOptions = {
+    pagination: false, // Disable default pagination.
```

Comment:
This field is already being set via an existing ACF field. we'll just need to change the ACF field to a true/false to return boolean values rather than strings

Reply:
updated slider.blade.php and splide.js file to implement ACF field group, slider__pagination, that shows/hides pagination at different breakpoints in commit 7a8dd48d77f9683e02904cbb06e5dc28970869e9h